### ISSUE TITLE
question about deprecation of this tool and alternative

### ISSUE THREAD

**Issue by StudioEtrange** [inconshreveable/gonative#33]
@inconshreveable 

Hello,

You post a comment on some other issue saying gonative should be considered as deprecated

>gonative is very, very old and almost certainly no longer necessary or the right way to do whatever it is that you're trying to do. i no longer use it and you should consider it a deprecated tool. 

For my information, what are you using now for having a cross compiling toolchain ? Or what is the right way to do ?

**Comment by inconshreveable** [comment]
The native Go toolchain supports cross-compilation sufficiently well for all of the usecases I need now.

**Comment by inconshreveable** [comment]
what is it exactly that you're trying to do?

**Comment by StudioEtrange** [comment]
2 years ago, I wrote simple scripts to cross compile go applications
i.e : ngrok go application built on multi platform : https://github.com/StudioEtrange/ngrok-build

I digged into the problem of cross compiling and I read your article here :
https://inconshreveable.com/04-30-2014/cross-compiling-golang-programs-with-native-libraries/
So I used gonative+gox by first installing go1.4 binaries

And everything fall apart with macos sierra/go1.4 bug.

But anyway,
It seems that your article is not relevant anymore if I understand you well ?

**Comment by inconshreveable** [comment]
yes, that's correct, for the most part it's not relevant anymore. there are still a few cases where that *might* matter to you, but they're even more rare than before and very likely not relevant to your use of ngrok

**Comment by StudioEtrange** [comment]
Ok
Thank you
